test/cors.test.js

```diff
@@ -252,10 +288,6 @@ test('Dynamic origin resolution (errored - promises)', t => {
   }
   fastify.register(cors, { origin })
 
-  fastify.get('/', (req, reply) => {
-    reply.send('ok')
-  })
```

Comment:
why was this deleted?

Reply:
It was never called. The test results in an error in the onRequest hook and the route never gets called. I removed it to try to make the tests consistent (remove unnecessary routes).